src/frontend/src/handler/show.rs

```diff
@@ -176,7 +175,7 @@ pub fn handle_show_create_object(
 
     Ok(PgResponse::new_for_stream(
         StatementType::SHOW_COMMAND,
-        Some(1),
```

Comment:
The row count should be 1? 🤔

Reply:
This field seems only used when `StatementType` is not a query. 🤔

https://github.com/risingwavelabs/risingwave/blob/8d082ecfc9d1c93bc7c6ef87dfeca60f381250d5/src/utils/pgwire/src/pg_response.rs#L131-L139

Reply:
Make sense. Please help to fix it in other handing parts of  non-queries.